### PR TITLE
Modify to use embed package instated of raw string literals

### PR DESCRIPTION
**Pull request by bells17**
resolve: https://github.com/topolvm/topolvm/issues/289
ref: https://github.com/topolvm/topolvm/pull/285#discussion_r603927908

### CODE REVIEW

File: e2e/testdata/e2e/node-capacity-pvc2.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+apiVersion: v1
+metadata:
+  name: topo-pvc3
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 8Gi
+  storageClassName: topolvm-provisioner
```
Comment (toshipp) [inline e2e/testdata/e2e/node-capacity-pvc2.yaml:35]: Would you add a new line?

File: e2e/testdata/hook/ephemeral-volume-pod-with-pvc.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            topolvm.cybozu.com/size: "2"
+      - name: my-ephemeral-volume2
+        csi:
+          driver: topolvm.cybozu.com
+          fsType: xfs
+          volumeAttributes:
+            topolvm.cybozu.com/size: "1"
+      - name: my-pvc-volume1
+        persistentVolumeClaim:
+          claimName: local-pvc1
```
Comment (toshipp) [inline e2e/testdata/hook/ephemeral-volume-pod-with-pvc.yaml:42]: Would you add a newline?

**Review by toshipp — CHANGES_REQUESTED** [review]

**Comment by llamerada-jp** [comment]
No additional comment. Would you reflect the comments by @toshipp

**Comment by bells17** [comment]
@toshipp @llamerada-jp Thank you for your review. I fixed the points of your comment.
Would you check this PR again?

**Review by toshipp — APPROVED** [review]
LGTM

**Review by llamerada-jp — APPROVED** [review]
LGTM

**Pull request merged**